php/lib/Ice.php

```diff
@@ -40,10 +40,10 @@
 $Ice_Encoding_1_0 = new Ice\EncodingVersion(1, 0);
 $Ice_Encoding_1_1 = new Ice\EncodingVersion(1, 1);
 
```

Comment:
As mentioned in the description, we have to pass both the sliceName and mappedName.
But, for these 4, obviously these names are identical.